test/unit/nft-staking.test.ts

```diff
@@ -160,6 +163,50 @@ describe('NFT Staking', () => {
             .unstakeNFTs(stakedNFTsV1)
             .should.be.revertedWith('Teller: not the owner of the NFT ID!')
         })
+
+        it('should be able to start to bridge NFTv2 to polygon', async () => {
+          const ownedNFTsV1 = await tellerNFTV1.getOwnedTokens(borrowerAddress)
+          await diamond.connect(borrower).mockStakeNFTsV1(ownedNFTsV1)
+          const stakedNFTsV1 = await diamond.getStakedNFTs(borrowerAddress)
```

Comment:
no need to mock stake V1 NFT first for this test. Actually, instead we should move this test to to the `poly-bridge.test.ts` file since we are testing the bridging function not staking